### PR TITLE
Update blog.json

### PR DESCRIPTION
**Pull request by adhirajpawar**
Please make sure to read the Contributing Guidelines:

https://github.com/neomjs/neo/blob/dev/CONTRIBUTING.md

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, _not_ the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by adhirajpawar** [comment]
hi @tobiu 
I’ve completed the steps outlined in issue #6012 and have added the blog post along with the required metadata. As this is my first contribution and I'm still learning the process, I hope everything is correct.

If I’ve made any mistakes or missed something, please do let me know—I’m eager to learn and fix anything that needs attention.

Thanks a lot for your patience and guidance throughout this contribution!

**Comment by tobiu** [comment]
word of advice: it is crucial to review your result visually before creating the PR.

take a look at: https://github.com/neomjs/neo/issues/6012
i added the  "How to add a new blog post into the Neo Website (Portal App)?" and "How to test the blog preview locally?" parts for a reason.

otherwise we can get results like this:
Screenshot 2024-10-04 at 21 06 37

while we could create a feature request ticket for the blog list to also support external URL based images, it would still look not nice:
Screenshot 2024-10-04 at 21 06 11

so i strongly recommend to search for a better cover image (e.g. browsing through the online examples and taking a screenshot).

 regarding the json properties for a blog item:
`id` 59, 60, 61, 6014 makes little sense => i would go for 62 :)
`publisher` as far as i know only the provider Medium supports it. when writing a blog post there, you can add it to a publisher like ITNEXT, Level Up Coding, JavaScript in Plain English and many many others (empty string in your case)
`selectedInto` is also an obsolete property for Medium blog posts => around 2 years ago, the team did promote high quality blog posts into specific categories to generate more traffic. sadly, they removed the feature, so now authors only get a "blog post got boosted" notification. empty array for other providers.
